tests/drivers/sensor/ina230/src/ina230_emul.c

```diff
@@ -151,22 +158,33 @@ static const struct i2c_emul_api ina230_emul_api_i2c = {
 	.transfer = ina230_emul_transfer_i2c,
 };
 
-#define INA230_EMUL(n)							\
-	static const struct ina230_emul_cfg ina230_emul_cfg_##n = {	\
-		.addr = DT_INST_REG_ADDR(n),				\
-	};								\
-	static struct ina230_emul_data ina230_emul_data_##n = {		\
-		.ina230_regs = {					\
-			{INA230_REG_CONFIG, 2, 0x4127},			\
-			{INA230_REG_SHUNT_VOLT, 2, 0},			\
-			{INA230_REG_BUS_VOLT, 2, 0},			\
-			{INA230_REG_POWER, 2, 0},			\
-			{INA230_REG_CURRENT, 2, 0},			\
-			{INA230_REG_CALIB, 2, 0},			\
-			{INA230_REG_MASK, 2, 0},			\
-			{INA230_REG_ALERT, 2, 0},			\
-		}};							\
-	EMUL_DT_INST_DEFINE(n, ina230_emul_init, &ina230_emul_data_##n, &ina230_emul_cfg_##n,	\
-			    &ina230_emul_api_i2c, NULL)
-
-DT_INST_FOREACH_STATUS_OKAY(INA230_EMUL)
+#define INA230_EMUL(n, v)                                                                          \
+	static const struct ina230_emul_cfg ina23##v##_emul_cfg_##n = {                            \
+		.addr = DT_INST_REG_ADDR(n),                                                       \
+	};                                                                                         \
+	static struct ina230_emul_data ina23##v##_emul_data_##n = {                                \
+		.ina230_regs =                                                                     \
+			{                                                                          \
+				{INA230_REG_CONFIG, 2, 0x4127},                                    \
+				{INA230_REG_SHUNT_VOLT, 2, 0},                                     \
+				{INA230_REG_BUS_VOLT, 2, 0},                                       \
+				{INA230_REG_POWER, 2, 0},                                          \
+				{INA230_REG_CURRENT, 2, 0},                                        \
+				{INA230_REG_CALIB, 2, 0},                                          \
+				{INA230_REG_MASK, 2, 0},                                           \
+				{INA230_REG_ALERT, 2, 0},                                          \
+			},                                                                         \
+		COND_CODE_1(is_236,                                                                \
```

Comment:
Where does `is_236` come from?

Reply:
Sorry about that, I was trying to use those registers only for INA236 for a moment.